tests/scripts/thread-cert/pktverify/layer_fields.py

```diff
@@ -64,10 +65,13 @@ def _auto(v: Union[LayerFieldsContainer, LayerField]):
     if ':' in dv and '::' not in dv and dv.replace(':', '') == rv:  # '88:00', '8800'
         return int(rv, 16)
 
-    if dv.endswith(' CST'):
+    if dv.startswith('Jan  '):
```

Comment:
why only process timestamp of `Jan` ? Can you give an example of timestamp string that you were trying to parse? 
@FiveDimensions

Reply:
use datetime.strptime to match timestamp format